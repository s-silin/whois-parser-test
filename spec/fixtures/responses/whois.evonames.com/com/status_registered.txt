
Domain Name: ENDINBLUE.COM
Registry Domain ID: 
Registrar WHOIS Server: whois.evonames.com
Registrar URL: https://evonames.com/
Updated Date: 2018-02-09 15:36:33.821851
Creation Date: 2017-08-08
Registrar Registration Expiration Date: 2018-08-08
Registrar: DANESCO TRADING LTD
Registrar IANA ID: 1418
Registrar Abuse Contact Email: abuse@evonames.com
Registrar Abuse Contact Phone: +357.95713635
Reseller: AHnames.com  https://www.AHnames.com/
Domain Status: ok
Registry Registrant ID: MR_11625649WP
Registrant Name: WhoisProtectService.net
Registrant Organization: PROTECTSERVICE, LTD.
Registrant Street: Agios Fylaxeos 66 and Chr. Perevou 2, Kalia Court, off. 601  
Registrant City: Limassol
Registrant State/Province: 
Registrant Postal Code: 3025
Registrant Country: Cyprus
Registrant Phone: +357.95713635
Registrant Phone Ext: 
Registrant Fax: +357.95713635
Registrant Fax Ext: 
Registrant Email: endinblue.com@whoisprotectservice.net
Registry Admin ID: MR_11625649WP
Admin Name: WhoisProtectService.net
Admin Organization: PROTECTSERVICE, LTD.
Admin Street: Agios Fylaxeos 66 and Chr. Perevou 2, Kalia Court, off. 601  
Admin City: Limassol
Admin State/Province: 
Admin Postal Code: 3025
Admin Country: Cyprus
Admin Phone: +357.95713635
Admin Phone Ext: 
Admin Fax: +357.95713635
Admin Fax Ext: 
Admin Email: endinblue.com@whoisprotectservice.net
Registry Tech ID: MR_11625649WP
Tech Name: WhoisProtectService.net
Tech Organization: PROTECTSERVICE, LTD.
Tech Street: Agios Fylaxeos 66 and Chr. Perevou 2, Kalia Court, off. 601  
Tech City: Limassol
Tech State/Province: 
Tech Postal Code: 3025
Tech Country: Cyprus
Tech Phone: +357.95713635
Tech Phone Ext: 
Tech Fax: +357.95713635
Tech Fax Ext: 
Tech Email: endinblue.com@whoisprotectservice.net
Registry Billing ID: MR_11625649WP
Billing Name: WhoisProtectService.net
Billing Organization: PROTECTSERVICE, LTD.
Billing Street: Agios Fylaxeos 66 and Chr. Perevou 2, Kalia Court, off. 601  
Billing City: Limassol
Billing State/Province: 
Billing Postal Code: 3025
Billing Country: Cyprus
Billing Phone: +357.95713635
Billing Phone Ext: 
Billing Fax: +357.95713635
Billing Fax Ext: 
Billing Email: endinblue.com@whoisprotectservice.net
Name Server: NS1.STORELAND.RU
Name Server: NS2.STORELAND.RU
Name Server: NS3.STORELAND.RU
Name Server: NS4.STORELAND.RU
DNSSEC: unsigned
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
>>> Last update of WHOIS database: 2018-02-09 15:37:10 <<<

Abuse email: abuse@ahnames.com

