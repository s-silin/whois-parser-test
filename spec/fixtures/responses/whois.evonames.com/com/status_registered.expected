#domain
  %s == "endinblue.com"

#domain_id
  %s == ""

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2017-08-08}

#updated_on
  %s %CLASS{time}
  %s %TIME{2018-02-09 15:36:33}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-08-08}

#registrar
  %s %CLASS{registrar}
  %s.id           == "1418"
  %s.name         == "DANESCO TRADING LTD"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "MR_11625649WP"
  %s[0].name          == "WhoisProtectService.net"
  %s[0].organization  == "PROTECTSERVICE, LTD."
  %s[0].address       == "Agios Fylaxeos 66 and Chr. Perevou 2, Kalia Court, off. 601"
  %s[0].city          == "Limassol"
  %s[0].zip           == "3025"
  %s[0].state         == ""
  %s[0].country_code  == "Cyprus"
  %s[0].phone         == "+357.95713635"
  %s[0].fax           == "+357.95713635"
  %s[0].email         == "endinblue.com@whoisprotectservice.net"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "MR_11625649WP"
  %s[0].name          == "WhoisProtectService.net"
  %s[0].organization  == "PROTECTSERVICE, LTD."
  %s[0].address       == "Agios Fylaxeos 66 and Chr. Perevou 2, Kalia Court, off. 601"
  %s[0].city          == "Limassol"
  %s[0].zip           == "3025"
  %s[0].state         == ""
  %s[0].country_code  == "Cyprus"
  %s[0].phone         == "+357.95713635"
  %s[0].fax           == "+357.95713635"
  %s[0].email         == "endinblue.com@whoisprotectservice.net"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "MR_11625649WP"
  %s[0].name          == "WhoisProtectService.net"
  %s[0].organization  == "PROTECTSERVICE, LTD."
  %s[0].address       == "Agios Fylaxeos 66 and Chr. Perevou 2, Kalia Court, off. 601"
  %s[0].city          == "Limassol"
  %s[0].zip           == "3025"
  %s[0].state         == ""
  %s[0].country_code  == "Cyprus"
  %s[0].phone         == "+357.95713635"
  %s[0].fax           == "+357.95713635"
  %s[0].email         == "endinblue.com@whoisprotectservice.net"


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.storeland.ru"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.storeland.ru"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.storeland.ru"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.storeland.ru"

